libs/api/domains/education/src/lib/graphql/frigg/option.model.ts

```diff
@@ -0,0 +1,15 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+
+import { FriggValueModel } from './value.model'
+
+@ObjectType()
+export class FriggOptionModel {
+  @Field(() => String)
```

Comment:
Don't need to explicitly provide the graphql type for string or boolean
[https://docs.nestjs.com/graphql/resolvers](link)

![image](https://github.com/island-is/island.is/assets/9449413/6180eeb6-27ee-4138-aa90-258337c66304)

Reply:
Fixed